symmetries.py

```diff
@@ -82,3 +83,47 @@ def randomize_symmetries_feat(features):
 def invert_symmetries_pi(symmetries, pis):
     return [apply_symmetry_pi(invert_symmetry(s), pi)
             for s, pi in zip(symmetries, pis)]
+
+
+def rotate_train(x, pi):
+    sym = tf.random_uniform(
+        [],
+        minval=0,
+        maxval=len(SYMMETRIES),
+        dtype=tf.int32,
+        seed=123)
+
+    def rotate(tensor):
```

Comment:
This is fricking cool.

Reply:
Yes, much better than what I had before :)